packages/web-react/src/components/Grid/__tests__/useGridItemStyleProps.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useGridItemStyleProps } from '../useGridItemStyleProps';
+
+describe('useGridItemStyleProps', () => {
+  it('should return defaults', () => {
+    const props = {};
+    const { result } = renderHook(() => useGridItemStyleProps(props));
+
+    expect(result.current.classProps).toBe('GridItem');
```

Comment:
I think it returns more than this prop. A few tests are missing IMHO.

Reply:
added

Reply:
Really…? 🤔